library/Smarty/demo/plugins/resource.mysqls.php

```diff
@@ -12,7 +12,7 @@
  *   `modified` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  *   `source` text,
  *   PRIMARY KEY (`name`)
- * ) ENGINE=InnoDB DEFAULT CHARSET=utf8;</pre>
+ * ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;</pre>
```

Comment:
Fixed in https://github.com/friendica/friendica/pull/3326/commits/08924b9c598fc969923fc16c3eccd90a931bbd1f